spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Graph.scala

```diff
@@ -599,13 +601,25 @@ trait GraphSerializable extends ContainerSerializable {
     (implicit ev: TensorNumeric[T]): AbstractModule[Activity, Activity, T] = {
     val (module, inputs, outputs, generateBackwardValue, sharedVariables) =
       prepareLoadModule(context)
-    if (generateBackwardValue != null) {
+    val attributes = module.getAttrMap
```

Comment:
looks like this line is useless

Reply:
it's used in line 615 below, so shall I merge it ?